pkg/server/routes.go

```diff
@@ -54,8 +54,12 @@ func (srv *RegistrationServer) SetupRoutes() error {
 		// /status is something you should always have in any of your services,
 		// please leave it as is.
 		healthService := health.NewHealthCheckService(srv.logger, srv.Config())
-		signupService := signup.NewSignupService(srv.logger, srv.Config())
-
+		signupService, err := signup.NewSignupService(srv.logger, srv.Config())
```

Comment:
don't you need to bind the service to an URL template, too?

Reply:
Yes, that is done a few lines down..

Reply:
ok, sorry, I missed that